src/lightkurve/lightcurve.py

```diff
@@ -3247,8 +3408,9 @@ def to_fits(
         }
 
         for kw in kepler_specific_data:
-            if ~np.asarray([kw.lower == k.lower() for k in extra_data]).any():
-                extra_data[kw] = kepler_specific_data[kw]
+            if hasattr(self, kw):
```

Comment:
This change in `to_fit()` appears to be unrelated. Some kind of merge problems?

Reply:
I don't know why this change ended up here. I will revert.